content/webapp/pages/seasons-beta/[id].tsx

```diff
@@ -0,0 +1,137 @@
+import { GetServerSideProps } from 'next';
+import { ReactElement } from 'react';
+import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
+import ContentPage from '@weco/common/views/components/ContentPage/ContentPage';
+import SeasonsHeader from '../../components/SeasonsHeader/SeasonsHeader';
+import { contentLd } from '@weco/common/utils/json-ld';
+import { removeUndefinedProps } from '@weco/common/utils/json';
+import Body from '@weco/common/views/components/Body/Body';
+import CardGrid from '@weco/common/views/components/CardGrid/CardGrid';
+import SpacingSection from '@weco/common/views/components/SpacingSection/SpacingSection';
+import SpacingComponent from '@weco/common/views/components/SpacingComponent/SpacingComponent';
+import { AppErrorProps } from '@weco/common/views/pages/_app';
+import {
+  getGlobalContextData,
+  WithGlobalContextData,
+} from '@weco/common/views/components/GlobalContextProvider/GlobalContextProvider';
+import { getServerData } from '@weco/common/server-data';
+import {
+  getSeason,
+  SeasonPrismicDocument,
+} from '../../services/prismic/seasons';
+import { api } from '../../services/prismic/api';
+import { Article } from '@weco/common/model/articles';
+import { Book } from '@weco/common/model/books';
+import { Event } from '@weco/common/model/events';
+import { Exhibition } from '@weco/common/model/exhibitions';
+import { Page } from '@weco/common/model/pages';
+import { ArticleSeries } from '@weco/common/model/article-series';
+import { Project } from '@weco/common/model/projects';
+import {
+  transformLabels,
+  transformMeta,
+} from '../../services/prismic/transformers';
+import { getImageUrlAtSize } from '../../services/prismic/images';
+import Image from 'components/Image/Image';
+
+type Props = {
+  season: SeasonPrismicDocument;
+  events?: Event[];
+  exhibitions?: Exhibition[];
+  articles?: Article[];
+  books?: Book[];
+  pages?: Page[];
+  articleSeries?: ArticleSeries[];
+  projects?: Project[];
+} & WithGlobalContextData;
+
+const SeasonPage = ({
+  season,
+
+  globalContextData,
+}: Props): ReactElement<Props> => {
+  const start = season.data.start ? new Date(season.data.start) : undefined;
+  const end = season.data.end ? new Date(season.data.end) : undefined;
+  const meta = transformMeta(season);
+  const labels = transformLabels(season);
+  const relatedContent = [];
+
+  return (
+    <PageLayout
+      title={meta.title}
+      description={meta.description ?? ''}
+      url={{ pathname: meta.url }}
+      jsonLd={contentLd(season)}
+      siteSection={'whats-on'}
+      openGraphType={meta.type}
+      imageUrl={meta.image && getImageUrlAtSize(meta.image, { w: 600 })}
+      // The type here is `string | undefined | null` as `alt: string | null`
+      // because prismic does actually return `null` and `image?: Image`.
+      // I'm okay with this here as it's at hte
```

Comment:
?

Reply:
really just tailed off didn't I - finished the comment.